SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 1.0.0
+  last-updated: '2023-10-20'
+  last-reviewed: '2023-10-20'
+  expiration-date: '2024-10-20T01:00:00.000Z'
+  project-url: https://github.com/jaegertracing/jaeger/
+  changelog: https://github.com/jaegertracing/jaeger/blob/main/CHANGELOG.md
+  license: https://github.com/jaegertracing/jaeger/blob/main/LICENSE
+project-lifecycle:
+  bug-fixes-only: false
+  core-maintainers:
+  - https://github.com/orgs/jaegertracing/teams/jaeger-maintainers
```

Comment:
not sure this is visible, maybe link to README#maintainers

Reply:
Yes, you at least need to be authenticated, so that's better, updated.